spec/index.bs

```diff
@@ -6736,10 +6741,20 @@ Issue: Because timestamp query provides high-resolution GPU timestamp, we need t
 interface GPUCanvasContext {
     GPUSwapChain configureSwapChain(GPUSwapChainDescriptor descriptor);
 
-    Promise<GPUTextureFormat> getSwapChainPreferredFormat(GPUDevice device);
+    GPUTextureFormat getSwapChainPreferredFormat(GPUAdapter adapter);
 };
 </script>
 
+{{GPUCanvasContext}} has the following internal slots:
+
+<dl dfn-type=attribute dfn-for=GPUCanvasContext>
+    : <dfn>\[[preferred_format]]</dfn>, of type [=ordered map=]&lt;{{GPUAdapter}}, {{GPUTextureFormat}}&gt;
```

Comment:
And this would be a map keyed off `adapter` instead of `GPUAdapter`

Reply:
Ditto